app/utils/email_tools.js

```diff
@@ -30,7 +30,22 @@ const validEmail = email => {
 }
 
 const commonTypos = email => {
-  mailcheck.defaultTopLevelDomains.push('gov.uk', 'org.uk')
+  mailcheck.defaultTopLevelDomains.push(
```

Comment:
This should be done at the top of the file, rather than in the function definition.

Reply:
then common typeos can just be `mailcheck.run` or even just `mailcheck` as this is a singleton